phys/module_physics_addtendc.F

```diff
@@ -714,12 +714,13 @@ SUBROUTINE phy_bl_ten(config_flags,rk_step,n_moist,n_scalar,     &
                 ims, ime, jms, jme, kms, kme,                    &
                 its, ite, jts, jte, kts, kte                     )
 
-        if (P_QT .ge. PARAM_FIRST_SCALAR)                                         &
-           CALL add_a2a(scalar_tendf(ims,kms,jms,P_QT),RQNIBLTEN,&
-               config_flags,                                    &
-               ids,ide, jds, jde, kds, kde,                     &
-               ims, ime, jms, jme, kms, kme,                    &
-               its, ite, jts, jte, kts, kte                     )
+!Note: QNC and QNI are updated in the pbl driver
+!        if (P_QT .ge. PARAM_FIRST_SCALAR)                                         &
+!           CALL add_a2a(scalar_tendf(ims,kms,jms,P_QT),RQNIBLTEN,&
+!               config_flags,                                    &
+!               ids,ide, jds, jde, kds, kde,                     &
+!               ims, ime, jms, jme, kms, kme,                    &
+!               its, ite, jts, jte, kts, kte                     )
```

Comment:
@joeolson42 If this is commented out, should it be just removed? The same for lines 748 - 754.